lit_gpt/lora.py

```diff
@@ -664,4 +672,4 @@ def merge_lora_weights(model: GPT) -> None:
     """Merge LoRA weights into the full-rank weights to speed up inference."""
     for module in model.modules():
         if isinstance(module, LoRALinear):
-            module.merge()
+            module.merge(verbose=False)
```

Comment:
A question: maybe it's better to use `verbose` as an argument in `merge_lora_weights`?
And then in generate script call `marge_lora_weights(model, verbose=False)`?

Reply:
I don't understand this: The usage in the LoRA generate script or the finetune script implies that the model has LoRA layers with r > 0 (enabled), so I don't see why the verbose flag is necessary.

Reply:
I'm sorry, should have been more explicit about it. It's because I've split large PR into smaller ones.
In another [PR](https://github.com/Lightning-AI/lit-gpt/pull/336/files#diff-9278d308e05826a4593d5fb4ddec59041a56291a309f856f4322c834c6fd1555R599) I use `LoRALinear` for layers with and without LoRA and it's possible by controlling rank value.
For example for the projection layer we can use `LoRALinear` with a correct rank value if we want to apply LoRA and we don't want - the same class but with rank of 0.
So when you call `merge_lora_weights` it will also call `merge` method for the projection layer and in this case it's fine that weights merging is not possible.